### ISSUE TITLE
[PE-D][Tester A] Running two find commands does not give expected results

### ISSUE THREAD

**Issue by nus-pe-script** [AY2425S1-CS2103T-W13-2/tp#221]
Expected second find command to find users based on the currently displayed list to users (ignore if this is the intended effect for find command)

Steps to reproduce
1) add n/John Doe p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25 t/friends t/owesMoney
2) add n/Betsy p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25 t/friends t/owesMoney
3) find n/Betsy
![Screenshot 2024-11-08 at 5.13.53 PM.png](https://raw.githubusercontent.com/CJianzhi/ped/main/files/74e224e8-9923-4415-81c6-cfb3a229488d.png)
4) find n/John

![Screenshot 2024-11-08 at 5.14.13 PM.png](https://raw.githubusercontent.com/CJianzhi/ped/main/files/2b2446e4-6d10-4ebf-afa9-ae546db727c7.png)

expected: No clients to be displayed since current list shown has only one person(Betsy)
actual: john doe is displayed

-------------
Labels: `type.FeatureFlaw` `severity.VeryLow`
original: CJianzhi/ped#7

**Comment by AppleJem** [comment]
Yes it is expected, find command searches from everyone, not just the currently displayed list of people. Will clarify in the UG
